content/learn/welcome/introduction.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Introduction"
+weight = 110
++++
+
+> The first thing you gotta discard, whenever you're learning about something, is your ego. - Luke Smith
+
+These are the learning resources for YAGPDB's powerful custom command system. This is your playground!
+
+We are going to teach you how to read code as well as how to write it. After all, how can you **write** if you cannot
+**read**? Learn to read, learn to solve problems, and learn to enjoy the process. The community is there to help, as
+long as you put in some effort yourself.
+
+To understand the custom commands interface you will be interacting with on the control panel, please read the
+documentation on [custom commands](/docs/custom-commands/commands).
+
+## General Tips
+
+As you start your journey with YAGPDB, there are a few things you should keep in mind. These tips will help you avoid
+common pitfalls and make your developing experience as a whole more enjoyable.
+
+You should start writing and testing custom commands on a separate testing server on Discord before adding them to your
+main server. This way, you already avoid a lot of problems caused by members potentially interfering with an incomplete
+custom command.
+
+There is no "undo"-button, and kicking YAGPDB won't reset your configuration. Write your changes down first, then you
+can revert them if something goes wrong.
+
+Never assume YAGPDB is running 24/7/365. It is a free service, and not guaranteed to be up all the time. Do not
+write your custom commands based on that assumption.
```

Comment:
"Never assume that YAGPDB is running 24/7/365: we do our best, but occasional downtime—for routine maintenance, upstream issues, and otherwise—is inevitable."

Premium exists, so the free service bit is not totally true. "Assumption" is also repeated.